### DISCUSSION TITLE
Training UO XGBoost %

### DISCUSSION THREAD

**Discussion by Onurbleacim** [kyleskom/NBA-Machine-Learning-Sports-Betting#424] [Q&A]
Hello everyone. Thank you for sharing this incredible project. One question, if anyone wants to help: I can only achieve a maximum of 54% accuracy with the XGBoost model UO. I've already adjusted certain parameters like the number of epochs, depth, etc., but it doesn't go beyond that. Is there anyone who has adjusted the parameters and wants to help me or give an idea for increase the performance? Thank you all in advance

**Comment by HyakuzaO** [comment]
I have models trained at 55.2%, but i didnt change nothing.

**Comment by nealmick** [comment]
gotta get more data or make other changes to how the data is formed. 

Just remember the accuracy of the model docent matter when it comes to betting.  can have 90% accuracy and your account will be limited....  there are easier ways to get limited, arbitrage or market consensus.

**Comment by fuzmik** [comment]
I question about training models. Do you guys edit the ./src/predict/xgboost.py to load out the new trained models? or mv the new created ones from ./models to ./models/xgboostmodels?

**Reply by HyakuzaO** [reply]
Move files to Xgboost folder and edit the names in src XgboostRunner
